docs/index.bs

```diff
@@ -2181,62 +2157,64 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
-        1. If |options|.{{CacheQueryOptions/cacheName}} is <a>present</a>, then:
-            1. Return a new <a>promise</a> |p| and run the following substeps <a>in parallel</a>:
-                1. [=map/For each=] |cacheName| → |cache| of <a>name to cache map</a>:
+        1. If |options|.{{CacheQueryOptions/cacheName}} is [=present=], then:
+            1. Return [=a new promise=] |promise| and run the following substeps [=in parallel=]:
+                1. [=map/For each=] |cacheName| → |cache| of the [=name to cache map=]:
                     1. If |options|.{{CacheQueryOptions/cacheName}} matches |cacheName|, then:
-                        1. Resolve |p| with the result of running the algorithm specified in {{Cache/match(request, options)}} method of {{Cache}} interface with |request| and |options| as the arguments (providing |cache| as thisArgument to the <a>\[[Call]]</a> internal method of {{Cache/match(request, options)}}.)
+                        1. Resolve |promise| with the result of running the algorithm specified in {{Cache/match(request, options)}} method of {{Cache}} interface with |request| and |options| (providing |cache| as thisArgument to the <a>\[[Call]]</a> internal method of {{Cache/match(request, options)}}.)
                         1. Abort these steps.
-                1. Resolve |p| with undefined.
+                1. Resolve |promise| with undefined.
         1. Else:
-            1. Let |p| be a <a>promise</a> resolved with undefined.
-            1. [=map/For each=] <var ignore>cacheName</var> → |cache| of <a>name to cache map</a>:
-                1. Set |p| to the result of <a>transforming</a> itself with a fulfillment handler that, when called with argument |v|, performs the following substeps <a>in parallel</a>:
-                    1. If |v| is not undefined, return |v|.
-                    1. Return the result of running the algorithm specified in {{Cache/match(request, options)}} method of {{Cache}} interface with |request| and |options| as the arguments (providing |cache|] as thisArgument to the <a>\[[Call]]</a> internal method of {{Cache/match(request, options)}}.)
-            1. Return |p|.
+            1. Let |promise| be [=a promise resolved with=] undefined.
+            1. [=map/For each=] <var ignore>cacheName</var> → |cache| of the [=name to cache map=]:
+                1. Set |promise| to the result of [=transforming=] itself with a fulfillment handler that, when called with argument |response|, performs the following substeps [=in parallel=]:
+                    1. If |response| is not undefined, return |response|.
+                    1. Return the result of running the algorithm specified in {{Cache/match(request, options)}} method of {{Cache}} interface with |request| and |options| as the arguments (providing |cache| as thisArgument to the `\[[Call]]` internal method of {{Cache/match(request, options)}}.)
```

Comment:
I don't think you need the slash in \[[Call]]

Reply:
Addressed.